code/src/reducers/tasks.js

```diff
@@ -0,0 +1,69 @@
+import { createSlice } from "@reduxjs/toolkit"
+import uniqid from 'uniqid'
+
+const tasks = createSlice({
+    name: 'tasks',
+    initialState: {
+        items: [
+            // {
+            //     id: 'fjhjf',
+            //     name: 'Do laundry',
+            //     isCaught: false
+            // },
+
+            // {
+            //     id: 'dsh',
+            //     name: 'Go shopping',
+            //     isCaught: false
+            // }
+        ],
+    },
+    reducers: {
+        addTask: (store, action) => {
+            const newTask = {
+                id: uniqid(),
+                text: action.payload,
+                isCaught: false,
```

Comment:
Just a recommendation to keep the store clean, we can pass these data directly in the onSubmitTask function, when we dispatch the input to the store. 😄